pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -573,8 +573,10 @@ type PodSandboxConfig struct {
 	// tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `protobuf:"bytes,7,rep,name=annotations" json:"annotations,omitempty" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
 	// Optional configurations specific to Linux hosts.
-	Linux            *LinuxPodSandboxConfig `protobuf:"bytes,8,opt,name=linux" json:"linux,omitempty"`
-	XXX_unrecognized []byte                 `json:"-"`
+	Linux *LinuxPodSandboxConfig `protobuf:"bytes,8,opt,name=linux" json:"linux,omitempty"`
+	// Custom sandbox ENV
+	PodInfraContainerEnv []*KeyValue `protobuf:"bytes,9,rep,name=podInfraContainerEnv" json:"podInfraContainerEnv,omitempty"`
```

Comment:
Here's a problem, mesos integration is using this env, and it replies on `options ...kubecontainer.Option` to set its value. This will require dockerservice implement `Runtime`. Any suggestions?